snap-app/src/solver/solver.js

```diff
@@ -1,23 +1,43 @@
 import React from "react";
 import { postJson } from "../fetch";
 import "./solver.css";
+import { withRouter } from "react-router-dom"
```

Comment:
alpha order and semicolon
```suggestion
import { withRouter } from "react-router-dom";
import { postJson } from "../fetch";
import "./solver.css";
```